website/src/components/Visualisations/IIGO/IIGO.tsx

```diff
@@ -1,15 +1,126 @@
 import React from 'react';
-import logo from '../../../assets/logo/logo512.png';
 import styles from './IIGO.module.css';
 import { OutputJSONType } from "../../../consts/types";
+import { RuleType, processRulesData } from "../IIGO/Util/ProcessedData";
+
+// UI imports
+import { makeStyles } from '@material-ui/core/styles';
+import Box from '@material-ui/core/Box';
+import Collapse from '@material-ui/core/Collapse';
+import IconButton from '@material-ui/core/IconButton';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import Typography from '@material-ui/core/Typography';
+import Paper from '@material-ui/core/Paper';
+import KeyboardArrowDownIcon from '@material-ui/icons/KeyboardArrowDown';
+import KeyboardArrowUpIcon from '@material-ui/icons/KeyboardArrowUp';
+
+import List from '@material-ui/core/List';
+import ListItemText from '@material-ui/core/ListItemText';
+
+const useRowStyles = makeStyles({
+    root: {
```

Comment:
can u change indentation to 2 spaces? i think the file was orig. 2 space

Reply:
hmm, I think the original files has 2 spaces indentation, but all the edited components switched to 4 space. Check Roles.
Therefore, I change all of mine to 4 spaces too.

Reply:
ok, we're setting up prettier soon so thats fine